subprojects/testing-base/src/integTest/groovy/org/gradle/testing/SecurityManagerIntegrationTest.groovy

```diff
@@ -60,7 +60,7 @@ public class SecurityManagerTest {
         // cause connection errors to show up in stderr.
         executer.withStackTraceChecksDisabled()
         fails('test')
-        failure.assertThatCause(matchesRegexp(".*Process 'Gradle Test Executor \\d' finished with non-zero exit value 1.*"))
+        failure.assertThatCause(matchesRegexp(".*Process 'Gradle Test Executor \\d?' finished with non-zero exit value 1.*"))
```

Comment:
```suggestion
        failure.assertThatCause(matchesRegexp(".*Process 'Gradle Test Executor \\d+' finished with non-zero exit value 1.*"))
```

I think we should use `\\d+` since `\\d?` can't match `Test Executor 11`.

Reply:
Good catch. I fixed it.